### ISSUE TITLE
WixClientServer error

### ISSUE THREAD

**Issue by hasirci** [safak/next-ecommerce#2]
![image](https://github.com/safak/next-ecommerce/assets/70998636/f6d1dc81-72b1-408b-90fd-b12792b8ba3d)

**Comment by kee236** [comment]
Ok

**Comment by Mrsiahkhanie** [comment]
> ![image](https://private-user-images.githubusercontent.com/70998636/334128211-f6d1dc81-72b1-408b-90fd-b12792b8ba3d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3MzgyNTEsIm5iZiI6MTcxODczNzk1MSwicGF0aCI6Ii83MDk5ODYzNi8zMzQxMjgyMTEtZjZkMWRjODEtNzJiMS00MDhiLTkwZmQtYjEyNzkyYjhiYTNkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE4VDE5MTIzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU3OTMyZjJlZjZhODk4OTdlNjFiZDhkOTJlYmY3ZGU4ODBmNmYzYjI5OWFlYTE2ODMyM2U0MmIzYzViNzFkNGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.dGLZkEENTLqXCDdTPZorjZMdfv9pJyzIB87280SMTPM)

I also had this problem, however, after this I see this error message. Do you have a solution?

![1](https://github.com/safak/next-ecommerce/assets/78175252/8b5a1c6f-8e18-4b99-b2dc-4c1b76ea4b6a)

**Comment by Mavaidos** [comment]
@Mrsiahkhanie  did you end up finding a solution ?

**Comment by Mrsiahkhanie** [comment]
> @Mrsiahkhanie did you end up finding a solution ?

Yes, the problem is probably from my system and internet, however, I uploaded the project from google idx and the problem was solved. I don't know the reason why the products are not displayed on the component server!

**Comment by HanzalaKhan07** [comment]
@hasirci same issue did you find the sollution????

**Comment by hasirci** [comment]
> @hasirci same issue did you find the sollution????

I didn't try again but i will look at for you soon and why this project owner not responding our issues so ?@safak

**Comment by HanzalaKhan07** [comment]
@hasirci Bro i remove "use client" from my main "page.tsx" file and now it is solved..
